server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -105,6 +105,8 @@ protected void start(String name, File rootDir, String configurationFile) {
          args.add("-Dcom.sun.management.jmxremote.authenticate=false");
          args.add("-Dcom.sun.management.jmxremote.ssl=false");
       }
+      args.add("-j");
+      args.add(System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_SERVER_CLUSTER_STACK));
```

Comment:
I think this will work from IntelliJ, as it picks up the properties from Maven into its run configurations, but still I'd like a default so that the driver works when the system property is not set.

I'd also move these lines up to replace `args.add("-Djgroups.bind.address=SITE_LOCAL");`, because they have more in common with `-b` than with the JMX system properties that follow, and because `-Djgroups.bind.address` after `-b` seems redundant.

Reply:
`-b SITE_LOCAL` is for the server bind address.
`-Djgroups.bind.address=SITE_LOCAL` is for the JGroups bind address.

Reply:
> I'd like a default so that the driver works when the system property is not set

Fixed